ctk/common/src/main/kotlin/org/codice/compliance/verification/core/SignatureSyntaxAndProcessingVerifier.kt

```diff
@@ -31,7 +30,7 @@ class SignatureSyntaxAndProcessingVerifier(private val node: Node) {
 
     /** 5.4.2 References */
     private fun verifySignatureSyntaxAndProcessing() {
-        node.children(SSOConstants.SIGNATURE).forEach {
+        node.recursiveChildren(SSOConstants.SIGNATURE).forEach {
```

Comment:
Out of curiosity, how did you notice this issue?

Reply:
@SmithJosh did